.github/workflows/ci.yaml

```diff
@@ -182,8 +193,14 @@ jobs:
                   path: |
                       ~/.cache/bazel
                       ~/.cache/bazel-repo
-                  key: bazel-cache-${{ hashFiles('**/BUILD.bazel', '**/*.bzl', 'WORKSPACE', '**/*.js') }}
-                  restore-keys: bazel-cache-
+                  key: >-
+                      bazel-cache-
+                      ${{ matrix.config }}-${{ matrix.bazelversion }}-
```

Comment:
leave out bazelversion and bzlmod from the key and make bazelisk have it's own cache.

Reply:
Nice one. Can you please put up a PR with the same change in the bazel-control rules-template? https://github.com/bazel-contrib/rules-template

Reply:
👍